src/UiFileupload.vue

```diff
@@ -48,6 +48,8 @@ import UiRippleInk from './UiRippleInk.vue';
 export default {
     name: 'ui-fileupload',
 
+    emits: ['update:modelValue', 'focus', 'blur', 'change'],
```

Comment:
I think we can keep the event here as just `input`. Since it can't be used with `v-model`, there's no expectation for it to be `update:modelValue`.

```suggestion
    emits: ['input', 'focus', 'blur', 'change'],
```

Reply:
Yes, been thinking about this, makes sense.

Reply:
Done!